### PR TITLE
Add --no-cache for building base images

### PR DESCRIPTION
**Pull request by gpcastro**
This is causing issues since we were not reinstalling the latest version of tools that go in the image.

### CODE REVIEW

File: build/build-runtimeimages-bases.sh [hunk 1]
```diff
@@ -85,7 +85,7 @@ for dockerFile in $dockerFiles; do
     cd $REPO_DIR
 
     echo
-    docker build -f $dockerFile -t $localImageTagName $labels .
+    docker build --no-cache -f $dockerFile -t $localImageTagName $labels .
```
Comment (kichalla) [inline build/build-runtimeimages-bases.sh]: As we discussed in our meeting, should we have this behind a flag so that we do it only when required?

Comment (arroyc) [reply build/build-runtimeimages-bases.sh]: I will like that to be adhoc

Comment (gpcastro) [reply build/build-runtimeimages-bases.sh]: I've changed the approach now, so only the steps we want to always run won't be cached.

File: images/runtime/node/4.4/Dockerfile.base [hunk 2]
```diff
@@ -33,6 +33,10 @@ RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-
   && tar -xJf "node-v$NODE_VERSION-linux-x64.tar.xz" -C /usr/local --strip-components=1 \
   && rm "node-v$NODE_VERSION-linux-x64.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt
 
+# This is a way to avoid using caches for the next stages, since we want the remaining steps
```
Comment (kichalla) [inline images/runtime/node/4.4/Dockerfile.base]: :clap:

**Pull request merged**